test/unit/visitor/defuse_analyze.cpp

```diff
@@ -315,6 +315,86 @@ SCENARIO("Perform DefUse analysis on NMODL constructs") {
         }
     }
 
+    GIVEN("local variable usage in else block") {
+        std::string nmodl_text = R"(
+            NEURON {
+                RANGE tau
+            }
+
+            DERIVATIVE states {
+                IF (tau == 1) {
+                } ELSE {
+                    LOCAL tau
+                    tau = 1 + tau
+                }
+            }
+        )";
```

Comment:
can we have this in the description of the issue or PR?

Reply:
Done